pkg/spoke/controllers/manifestwork_controller.go

```diff
@@ -411,48 +411,28 @@ func buildManifestResourceMeta(index int, object runtime.Object, restMapper *res
 		Ordinal: int32(index),
 	}
 
-	if object == nil {
+	if reflect.ValueOf(object).IsNil() {
```

Comment:
I think we still need to check if object==nil before this? I think reflect.ValueOf(nil) will panic.

Reply:
Yes, you are right. Code is updated.